src/lib/components/Footer/Footer.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Footer.css';
+
+const Footer = (props) => {
+  const centered = {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  };
+
+  return (
+    <footer style={props.centered ? centered : null} className="p-footer">
```

Comment:
maybe it's a good way of doing in react, but I would put the `centered` var in the css file and write this line something like this:
```html
<footer className="p-footer {props.centered ? 'centered' : null} ">
```

in the css:
```css
.centered {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
}
```